### PR TITLE
Add create button and page

### PR DESCRIPTION
**Pull request by gracenng**
First PR in a while, let me know!

### CODE REVIEW

File: mobile/src/src/components/Create/CreatePage.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { StyleSheet, View } from "react-native";
+import Header from "../common/Header";
+
+
```
Comment (trentstauff) [inline mobile/src/src/components/Create/CreatePage.jsx:5]: delete extra white space

File: mobile/src/src/components/Create/CreatePage.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+const CreatePage = () => {
+  return (
+    <View style={styles.container}>
+      <Header title="Create" />
+
```
Comment (trentstauff) [inline mobile/src/src/components/Create/CreatePage.jsx:10]: delete line

File: mobile/src/src/components/Create/index.jsx [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import CreatePage from "./CreatePage";
+
+export default CreatePage;
```
Comment (trentstauff) [inline mobile/src/src/components/Create/index.jsx:3]: add newline at end of file

File: mobile/src/src/components/Create/CreatePage.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+};
+
+export default CreatePage;
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: "#eeeeee",
+  },
+});
```
Comment (trentstauff) [inline mobile/src/src/components/Create/CreatePage.jsx:22]: newline end of file

File: mobile/src/src/navigators/TabNavigator.jsx [hunk 5]
```diff
@@ -51,26 +52,35 @@ const TabNavigator = ({ navigation }) => {
...
+          tabBarIcon: ({ color}) => (
+            <MaterialCommunityIcons
+              name="plus-circle-outline"
+              color={color}
+              size={24}
+            />
           ),
         }}
       />
+      {/* <LinearGradient
```
Comment (trentstauff) [inline mobile/src/src/navigators/TabNavigator.jsx:69]: Do you need this comment?

File: mobile/src/src/navigators/TabNavigator.jsx [hunk 6]
```diff
@@ -51,26 +52,35 @@ const TabNavigator = ({ navigation }) => {
...
+      
       <Tab.Screen
-        name="Housing"
-        component={HousingPage}
+        name="Create"
+        component={CreatePage}
         options={{
-          tabBarIcon: ({ color }) => (
-            <MaterialCommunityIcons name="home" color={color} size={24} />
+          tabBarIcon: ({ color}) => (
```
Comment (trentstauff) [inline mobile/src/src/navigators/TabNavigator.jsx:60]: As you can see, ({ color}) isnt centered. Im going to stop the review now, install the prettier extension and just go through your files youve touched and do ctrl+S, and upload the changes

Comment (trentstauff) [reply mobile/src/src/navigators/TabNavigator.jsx:60]: Other than that- i havent tested the code locally yet, so i will do that and let you know. Code looks good though grace, good work.

**Review by trentstauff — CHANGES_REQUESTED** [review]
Just install prettier and clean code up- i also need to test it after so dont merge yet

**Comment by gracenng** [comment]
image
 tested this morning after re-cloning

**Review by trentstauff — APPROVED** [review]
Looks good, nice work Grace.

**Pull request merged**